src/Phinx/Db/Adapter/AdapterWrapper.php

```diff
@@ -349,6 +349,22 @@ public function createTable(Table $table, array $columns = [], array $indexes =
         $this->getAdapter()->createTable($table, $columns, $indexes);
     }
 
+    /**
+     * {@inheritdoc}
+     */
+    public function changePrimaryKey(Table $table, $newColumns)
+    {
+        $this->getAdapter()->changePrimaryKey($table, $newColumns);
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function changeComment(Table $table, $newComment)
+    {
+        $this->getAdapter()->changeComment($table, $newComment);
```

Comment:
This method and the one above should be guarded by an `if` condition, checking that the adapter is `DirectActionInterface`

Reply:
I'm actually considering removing them from AdapterWrapper for consistency, since now that I'm looking at the difference between this class and DirectActionInterface, I'm noticing that this class doesn't have any of the other table modifications anyway.
Do you agree or should I strive to keep them as suggested?

Reply:
I agree!